md/project_deploy_in_studio.md

```diff
@@ -0,0 +1,36 @@
+# Project deployment in a Studio
+
+This page presents an overview of the _Deploy wizard_ capabilities in the Studio
+
+## Artifact selection
+
+After importing or cloning a project in the Studio, you will have the possibility to directly deploy your project in the embedded studio runtime.  
+You can open the _Deploy wizard_ using the contextual menu on the project in the Project Explorer view or using find the action directly in the toolbar bar.  
```

Comment:
```suggestion
You can open the _Deploy wizard_ using the contextual menu of the project in the Project Explorer view or find the action directly in the toolbar.  
```

Reply:
what toolbar? we still call it the "Coolbar".
Or you need to replace it everywhere by "the main toolbar", as we have contextual toolbars in some editors.

Reply:
We can't do that. It is a toolbar